spec/features/visit_home_page_spec.rb

```diff
@@ -51,7 +51,14 @@
     end
   end
 
-  context "shows the correct date-dependent banners" do
+  scenario "showing the ICE warning banner" do
+    visit "/"
+
+    expect(page).to have_text "Immigrant filers: As of April 7th, the IRS and Dept. of Homeland Security (including ICE) have reached an agreement to share data between the agencies. Please see our FAQ for more information."
+    expect(page.all(:css, '.slab--banner').length).to eq 1
+  end
+
+  xcontext "shows the correct date-dependent banners" do
```

Comment:
i expect we'll want to incorporate the date-dependent info somewhere in a later story?

Reply:
yeah i wasn't sure whether to nuke it entirely and since this is a last minute story i feel okay about skipping some tests for now